app/views/support_interface/performance/feature_metrics_dashboard.html.erb

```diff
@@ -408,6 +408,32 @@
         </div>
       </div>
     </div>
+    <div class="govuk-grid-column-one-half">
+      <div id="headline-stat" class="govuk-!-margin-bottom-4">
```

Comment:
Can we remove this `id`? There should only be one unique `id` per page, and I note that this is being repeated for each set of stats. I’ll make a separate PR to address this (and simplify the markup a bit, too)

```suggestion
      <div class="govuk-!-margin-bottom-4">
```